src/backend/joanie/settings.py

```diff
@@ -394,7 +394,7 @@ class Base(Configuration):
         ),
     }
     JOANIE_PAYMENT_SCHEDULE_LIMITS = values.DictValue(
-        {150: (100,), 200: (30, 70), 500: (30, 35, 35), 1000: (30, 25, 25, 20)},
+        {200: (30, 70), 500: (30, 35, 35), 1000: (30, 25, 25, 20)},
```

Comment:
If I remember correctly, we should have `0` as the first payment step.
```suggestion
        {0: (30, 70), 500: (30, 35, 35), 1000: (30, 25, 25, 20)},
```

Reply:
This [method](https://github.com/openfun/joanie/blob/6efa9d849bd36146bfb42d06db7fe60a36eaaf80/src/backend/joanie/core/utils/payment_schedule.py#L28) checks whether the amount is less or equal to the limit defined in the configurations